examples/ocaml_examples.ml

```diff
@@ -42,4 +42,4 @@ let _ =
                    (Js.Json.stringify (Js.Json.object_ payload)))
           ~headers:(Fetch.HeadersInit.make
                       ([%bs.obj { Content-Type = "application/json" }])) ()))
-      |> (then_ Fetch.Response.json)
\ No newline at end of file
+      |> (then_ Fetch.Response.json) *)
```

Comment:
I should add a dict API to fix this properly, and also set up a simple "does it compile" CI to ensure this doesn't happen again. Might get around to doing that before merging this, so I don't forget and to reduce the noise in this PR.